src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -454,8 +454,8 @@ QueryNetworkResult Plugin::QueryNetwork(const CNNNetwork& network,
         }
     }
 
-    for (auto&& layerName : supported) {
-        if (InferenceEngine::details::contains(unsupported, layerName)) {
+    for (auto&& layerName : unsupported) {
+        if (InferenceEngine::details::contains(supported, layerName)) {
```

Comment:
am I right that logic is the same, but we fix SEGFAULT which was caused that we iterate over container which is changed inside the loop?

Reply:
yeap, you're right.
next layerName after erase call was invalid